RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -1367,14 +1365,6 @@ boolean L11_hiddenCity()
 			if(shouldForceElevatorAction)
 			{
 				elevatorAction = auto_forceNextNoncombat();
-
-				if(in_pokefam())
-				{
-					if(get_property("relocatePygmyLawyer").to_int() != my_ascensions())
-					{
-						return autoAdv($location[The Hidden Apartment Building]);
-					}
-				}
```

Comment:
why was this removed?
it sounds like in pokefam the priority should be to do this first rather than not doing it at all.

Reply:
There’s no way to obtain thrice cursed in pokefam, other than the cursed punch.  My thought on routing was that going through an entire first round of the apartment building to relocate the lawyers and then have a second round going to fight the spectre would be less effective than just going through the first round to fight the spectre.   I’m happy to add back in if you or anyone thinks differently.

Reply:
why is thrice cursed not working without punch in pokefam?
also I think the idea here is not relocating them for the apartment, it is relocating them for the office by going to the apartment to relocate them.

Reply:
Initial combat actions (like receiving the curse from a shaman) do not work in pokefam, unfortunately.

Reply:
> Initial combat actions (like receiving the curse from a shaman) do not work in pokefam, unfortunately.

I see now. thank you for clarifying. So in that case the only options are:
1. drink curse punch
2. get the NC and choose the option that gives +1 curse.

hmmm. so the question is, is getting rid of the lawyer for the office worth spending another 8 adv on the apartment building... you cannot banish, but I think you could get all the accountants done easily without it... so yea, overall I think it is a negative feature. Deleting it is the correct choice

Reply:
Thanks again taltamir